sqlengine/postgres_engine.go

```diff
@@ -317,6 +317,22 @@ func (d *PostgresEngine) JDBCURI(address string, port int64, dbname string, user
 	return fmt.Sprintf("jdbc:postgresql://%s:%d/%s?%s", address, port, dbname, params.Encode())
 }
 
+const createExtensionPattern = `CREATE EXTENSION IF NOT EXISTS "{{.extension}}"`
+
+func (d *PostgresEngine) CreateExtensions(extensions []string) error {
+	for _, extension := range extensions {
+		createExtensionTemplate := template.Must(template.New(extension + "Extension").Parse(createExtensionPattern))
```

Comment:
To gather the reviewer's opinion: I used a template here for consistency with the other SQL statements, but given the simplicity of this statement I am wondering if the standard SQL placeholders in the `database/sql` package would have been more appropriate.